sdk/servicebus/service-bus/package.json

```diff
@@ -55,7 +55,7 @@
     "build:types": "downlevel-dts types/latest types/3.1",
     "build": "tsc -p . && rollup -c 2>&1 && npm run extract-api && npm run build:types",
     "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-esm test-dist types *.tgz *.log coverage coverage-browser .nyc_output",
+    "clean": "rimraf dist* dist-esm dist-test test-dist types *.tgz *.log coverage coverage-browser .nyc_output",
```

Comment:
Just curious, why did you change `dist` to `dist*`? Feels a little weird since you explicitly list dist-esm and dist-test.

Reply:
Hmmm, overlooked.
First added the dist-test... and then saw another dist-browser, so added a *.

Reply:
Oops I should have waited before merging - @HarshaNalluru